From: Philipp Hahn <hahn@univention.de>
Date: Wed, 15 Jul 2015 17:53:59 +0200
Subject: Skip downloading xrdp GIT
Organization: Univention GmbH, Bremen, Germany

Don't download the repository again if it already exists.
---
 X11rdp-o-matic.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/X11rdp-o-matic.sh b/X11rdp-o-matic.sh
index c4f7f06..b3ef7ec 100755
--- a/X11rdp-o-matic.sh
+++ b/X11rdp-o-matic.sh
@@ -327,12 +327,14 @@ install_package_interactive()
 
 download_xrdp_interactive()
 {
+  [ -d xrdp ] ||
   git clone --depth 1 "$XRDPGIT" -b "$XRDPBRANCH" 2>&1 | dialog  --progressbox "Downloading xrdp source..." 30 100
 }
 
 download_xrdp_noninteractive()
 {
   echo "Downloading xrdp source from the GIT repository..."
+  [ -d xrdp ] ||
   git clone --depth 1 "$XRDPGIT" -b "$XRDPBRANCH"
 }
 
